win/pkgs/utf8proc.ps1

```diff
@@ -0,0 +1,69 @@
+#Requires -RunAsAdministrator
+
+Get-Content "$PSScriptRoot/utils/re-entry.ps1" -Raw | Invoke-Expression
+$ErrorActionPreference="Stop"
+
+. "$PSScriptRoot/env/mirror.ps1"
+. "$PSScriptRoot/env/toolchain.ps1"
+
+pushd ${Env:TMP}
+$repo="${Env:GIT_MIRROR}/JuliaStrings/utf8proc.git"
+$proj="$($repo -replace '.*/','' -replace '.git$','')"
+$root="${Env:TMP}/$proj"
+
+rm -Force -Recurse -ErrorAction SilentlyContinue -WarningAction SilentlyContinue "$root"
+if (Test-Path "$root")
+{
+    echo "Failed to remove `"$root`""
+    Exit 1
+}
+
+$latest_ver='v' + $($(git ls-remote --tags "$repo") -match '.*refs/tags/v[0-9\.]*$' -replace '.*refs/tags/v','' | sort {[Version]$_})[-1]
+git clone --depth 1 --single-branch -b "$latest_ver" "$repo"
+pushd "$root"
+mkdir build-win
+pushd build-win
+
+# - Please ignore warning C4273 since it is by design and safe.
+#   dllexport in ".cc" will precedence over dllimport in ".h".
+cmake                                                               `
+    -DBUILD_SHARED_LIBS=ON                                          `
+    -DCMAKE_BUILD_TYPE=Release                                      `
+    -DCMAKE_C_FLAGS="/GL /MP /Zi"                                   `
+    -DCMAKE_CXX_FLAGS="/EHsc /GL /MP /Zi"                           `
+    -DCMAKE_EXE_LINKER_FLAGS="/DEBUG:FASTLINK /LTCG:incremental"    `
+    -DCMAKE_INSTALL_PREFIX="${Env:ProgramFiles}/utf8proc"               `
+    -DCMAKE_PDB_OUTPUT_DIRECTORY="${PWD}/pdb"                       `
+    -DCMAKE_SHARED_LINKER_FLAGS="/DEBUG:FASTLINK /LTCG:incremental" `
+    -DCMAKE_STATIC_LINKER_FLAGS="/LTCG:incremental"                 `
+    -G"Ninja"                                                       `
+    ..
+
+cmake --build .
+if (-Not $?)
+{
+    echo "Failed to build."
+    echo "Retry with best-effort for logging."
+    echo "You may Ctrl-C this if you don't need the log file."
+    cmake --build . -- -k0
+    cmake --build . 2>&1 | tee ${Env:TMP}/${proj}.log
+    exit 1
+}
+
+$ErrorActionPreference="SilentlyContinue"
+cmake --build . --target test
+if (-Not $?)
+{
+    echo "Check failed but we temporarily bypass it."
```

Comment:
Only use this for unstable UT.
If utf8proc has stable UT, fail it here by removing this if block and `$ErrorActionPreference` around.

Reply:
removed this as utf8proc has stable ut